docs/series/system/SystemSeriesA03.md

```diff
@@ -0,0 +1,72 @@
+# A03 ❯ NVIDIA Driver Installation Guide
+<small>ℹ️ This article is part of AlmaLinux [System Series](/series/).</small>
+<hr>
+| 💡 | Experience Level  | ⭐☆☆☆☆ |
+|--- | --------- | --------|
+| 📆 | <small>Last modified </small>| 2023-05-22
+| 🔧 | <small>Tested by <br> ↳ version \| platform \| date </small>| <small>[none](mailto:none@almalinux.org) <br>  ↳ 9.x \| x86_64 \| 2023-05-xx </small>|
+<br>
+
+
+## 🌟 Introduction
+
+This guide provides instructions how to install NVIDIA's graphics driver for AlmaLinux in to variants:
+
+- **Binary Driver** [**8.x**](SystemSeriesA03R8.md) | [**9.x**](SystemSeriesA03R9.md) 👈 <small>USING PRECOMPILED DRIVERS IS RECOMMENDED</small>
+- **Compiling From Source** [**8.x**](SystemSeriesA03R8.md) | [**9.x**](SystemSeriesA03R9.md) ⚠️  <small>REQURIES SWITCHING TO ELRepo KERNEL</small>
+
+## 🧠 Fundamental Concepts
+
+### Dynamic Kernel Module Support (DKMS)
+
+DKMS is a system utility that allows you to automatically build kernel modules.
+
+### ELRepo Kernel
+
+ELRepo is a community repository for Enterprise Linux distributions (e.g., RHEL, CentOS, AlmaLinux). It focuses on hardware-related packages to enhance your Linux experience. This is repo provides a kernel (packages) which tracks mainline kernel very closely. In result, this kernel is updated frequently and sometimes it leads to temporary incompatiblity with NVIDIA driver sources.
```

Comment:
ELRepo is a community repository for Enterprise Linux distributions (e.g., RHEL, CentOS, AlmaLinux). It focuses on hardware-related packages to enhance your Linux experience. This repo provides a kernel (packages) that closely tracks the mainline kernel. As a result, this kernel is updated frequently and sometimes leads to temporary incompatiblity with NVIDIA driver sources.